apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-applications/vo-applications.component.ts

```diff
@@ -94,6 +108,25 @@ export class VoApplicationsComponent implements OnInit {
     }
   }
 
+  dateChange() {
+    this.dateObject = {startDate: this.startDate.value, endDate: this.endDate.value};
+    localStorage.setItem("dateObjectVo" + this.vo.id, JSON.stringify(this.dateObject));
```

Comment:
so the problem with this is that,  when I open this page on 2.2.2020 and change the start date to something(doesnt matter) then the start and end date will be saved to local host
problem will happen when I open this page on 3.2.2020(or any day after the 2.2.2020) and the end day will still be 2.2.2020, so if somebody made new application on 3.2 then the user will not see it - and you can bet that he/she will not notice the end date
I think the end date shouldn't be cached into the localStorage and I think that if the start date should be cached is questionmark, maybe it would be best if you ask Pavel Vyskocil/our support if the start date should be cached

Reply:
I discussed it with Pavel Vyskočil and in his opinion this dates shouldn't be cached, so I removed date storing in localStorage. Due to these changes I was able to simplify the code and remove dateChange() function in vo and group applications components. This function was replaced by already existing function select() in html.